bcipy/task/data.py

```diff
@@ -6,6 +6,21 @@
 EVIDENCE_SUFFIX = "_evidence"
 
 
+def rounded(values: List[float], precision: int) -> List[float]:
```

Comment:
The type signature for this is off if precision can be `None`. I think this may be more straightforward if we enforce int for precision.. either forcing the upstream callers to check whether it should be rounded at all or have a sufficient default value `precision: int == 10`.